code/src/Artist.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './artist.css';
+
+const Artist = props => {
+  if (props.index === 0) {
+    return (
+      <a className="artist__name" href={props.artistURL}>
+        {props.artistName}
+      </a>
+    );
+  } else if (props.index === props.length - 1) {
+    return (
+      <a className="artist__name" href={props.artistURL}>
+        {'\u00A0& ' + props.artistName}
```

Comment:
In general great job. Nonetheless, it could be improved - for now, if we hover over the link, both link text and `,` or `&` are underlined. What you could do is wrapping separator mark with another tag, like this :

```
return (
   <>
      <span>\u00A0&</span>
      <a className="artist__name" href={props.artistURL}>
        {props.artistName}
      </a>
   </>
    );
```